### ISSUE TITLE
New ads on Tumblr dashboard

### ISSUE THREAD

**Issue by bjornstar** [bjornstar/Tumblr-Savior#25]
![screenshot from 2015-04-16 16 41 07](https://cloud.githubusercontent.com/assets/20630/7176513/133183ea-e45a-11e4-9e74-688143c3405e.png)

Here's the html for it.
``` html

<li class="notification single_notification alt takeover-container mb_tracked" data-pt="eyJpIjoiTmZGdGVNQVRKQnNmTmNaVGw5OTkiLCJ2IjoxLCJ0IjoxNDI5MTY5MDM3LCJjIjpbXSwiZCI6eyJ0Ijp7ImwiOjMxLCJ0IjoiMjAifX0sImgiOiIzYWVlYTZkMWI2In0=" data-view-exists="true">
  <a href="/explore/betterforit" class="takeover-banner-link">
    <div class="stretchy_kids">
      <div class="stretchy_kid_container">
        <div class="banner_notification_inner stretchy_kid clearfix">
          <div class="notification_sentence">
            <span class="banner_text">Embrace your uncomfort zone</span>
          </div>
        </div>
      </div>
      <div class="control banner_avatar_frame">
        <img alt="" class="avatar" id="notification_avatar_4szbf68cuasauat" src="https://33.media.tumblr.com/avatar_818fdc8a235b_128.png" data-src="https://33.media.tumblr.com/avatar_818fdc8a235b_128.png">
        <div class="avatar_glass"></div>
      </div>
      <div class="stretchy_kid_container">
        <div class="notification_follow stretchy_kid">
          <div class="post_control post_control_menu sponsor" title="Sponsor Options">
            <div class="sponsored_badge mini">
              <div class="sponsored_badge_icon">
                <span class="dollar_sign">$</span>
                <span class="ring" style="transform: rotate(0deg);"></span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="stretchy_kid_container">
      <i class="icon icon_arrow_carrot_right"></i>
    </div>
  </a>
</li>
```

**Comment by bjornstar** [comment]
Here's what a normal notification looks like:

``` html
<li class="notification single_notification alt notification_reblog can_ignore" data-view-exists="true">
	<div class="notification_inner  clearfix">
		<div class="notification_sentence">
			<div class="hide_overflow">
				<a class="username" href="http://comicsworkbook.tumblr.com/">comicsworkbook</a>
 				<a class="notification_target" href="http://comicsworkbook.tumblr.com/post/116558965417/comicsworkbook-bjornstar-oh-tumblr-when">reblogged</a>your photo<span class="colon">:</span>
				<em style="white-space:nowrap;">
					<a href="http://comicsworkbook.tumblr.com/post/116558965417/comicsworkbook-bjornstar-oh-tumblr-when">Oh tumblr, when will you stop sticking weird stuff...</a>
				</em>
			</div>
			<blockquote>Check out Tumblr Savior if you dont have it</blockquote>
		</div>
	</div>
	<a target="_blank" href="http://comicsworkbook.tumblr.com/" data-tumblelog-popover="{&quot;avatar_url&quot;:&quot;https:\/\/38.media.tumblr.com\/avatar_e2e14872c2af_128.png&quot;,&quot;dashboard_url&quot;:&quot;\/blog\/comicsworkbook&quot;,&quot;url&quot;:&quot;http:\/\/comicsworkbook.tumblr.com&quot;,&quot;name&quot;:&quot;comicsworkbook&quot;,&quot;cname&quot;:&quot;&quot;,&quot;description&quot;:&quot;An online magazine for Comic Book Makers&quot;,&quot;description_sanitized&quot;:&quot;An online magazine for Comic Book Makers&quot;,&quot;title&quot;:&quot;Comics Workbook&quot;,&quot;likes&quot;:false,&quot;is_private&quot;:false,&quot;is_group&quot;:true,&quot;customizable&quot;:false,&quot;following&quot;:false,&quot;premium_partner&quot;:false,&quot;global_theme_params&quot;:{&quot;avatar_shape&quot;:&quot;square&quot;,&quot;background_color&quot;:&quot;#FAFAFA&quot;,&quot;body_font&quot;:&quot;Helvetica Neue&quot;,&quot;header_bounds&quot;:0,&quot;header_image&quot;:&quot;http:\/\/assets.tumblr.com\/images\/default_header\/optica_pattern_04.png?_v=7c4e5e82cf797042596e2e64af1c383f&quot;,&quot;header_image_dimens&quot;:&quot;&quot;,&quot;header_image_focused&quot;:&quot;https:\/\/secure.assets.tumblr.com\/images\/default_header\/optica_pattern_04_640.png?_v=7c4e5e82cf797042596e2e64af1c383f&quot;,&quot;header_image_scaled&quot;:&quot;http:\/\/assets.tumblr.com\/images\/default_header\/optica_pattern_04.png?_v=7c4e5e82cf797042596e2e64af1c383f&quot;,&quot;header_stretch&quot;:true,&quot;link_color&quot;:&quot;#529ECC&quot;,&quot;show_avatar&quot;:true,&quot;show_description&quot;:true,&quot;show_header_image&quot;:true,&quot;show_title&quot;:true,&quot;title_color&quot;:&quot;#444444&quot;,&quot;title_font&quot;:&quot;Gibson&quot;,&quot;title_font_weight&quot;:&quot;bold&quot;}}" class="control avatar_frame">
		<img alt="" class="avatar" id="notification_avatar_6e8h2uzs4hms6eu" src="https://33.media.tumblr.com/avatar_e2e14872c2af_40.png" data-src="https://33.media.tumblr.com/avatar_e2e14872c2af_40.png">
		<div class="avatar_glass"></div>
	</a>
	<div class="control notification_right">
		<a target="_blank" class="preview_frame" href="http://bjornstar.tumblr.com/post/116540875656/oh-tumblr-when-will-you-stop-sticking-weird-stuff">
			<img src="https://41.media.tumblr.com/e86d9d84d62bde629d08a60d913f58f8/tumblr_nmw2puHshC1qz5vceo1_75sq.png">
		</a>
		<a href="#" title="ignore comicsworkbook" data-tumblelog="comicsworkbook" class="block">ignore</a>
	</div>
</li>
```

Make sure you don't hide normal notifications too.

**Comment by bjornstar** [comment]
Fixed in v0.4.21 - https://github.com/bjornstar/Tumblr-Savior/commit/ac0e0f21a0a8b1a8d9d5c344b732b0662e8a972d

**Comment by jristz** [comment]
and is gone without tumblr-savior.
Aparently this could happend or not but is better safer than sorry. thanks.
